tests/user_commands/tests.py

```diff
@@ -401,6 +402,8 @@ def test_subparser_dest_required_args(self):
 
     def test_subparser_invalid_option(self):
         msg = "invalid choice: 'test' (choose from 'foo')"
+        if PY314:
+            msg = "Error: argument {foo}: invalid choice: 'test' (choose from foo)"
```

Comment:
I'd assert only the same part of the error message as for Python < 3.14.
```suggestion
            msg = "invalid choice: 'test' (choose from foo)"
```